hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegion.java

```diff
@@ -715,7 +715,8 @@ void sawNoSuchFamily() {
   // Used to track interruptible holders of the region lock. Currently that is only RPC handler
   // threads. Boolean value in map determines if lock holder can be interrupted, normally true,
   // but may be false when thread is transiting a critical section.
-  final ConcurrentHashMap<Thread, Boolean> regionLockHolders;
+  final ConcurrentHashMap<Thread, Boolean> intrRegionLockHolders;
```

Comment:
I agree with @virajjasani that we probably don't need two maps here, and we do seem to be losing some readability advantages.

Two options:
1. `isOperationInterruptible` is effectively static based on the op... so inside of `enableInterrupts` or `disableInterrupts` methods, we can also only take action `if isOperationInterruptible`.
2. If we think that somehow an op could be interruptible, perhaps in the future, based on dynamic properties of the op... then we could create a class (or pair) to track two booleans - one for whether to interrupt, and one for whether to ever interrupt. Then, similar to (1) above, we can use that other boolean to determine whether to `enableInterrupts` or `disableInterrupts`.